app/views/_pod-template.html

```diff
@@ -7,194 +7,67 @@
-                    authored by {{build.spec.revision.git.author.name}}
-                  </span>
-                </span>
-                <span ng-if="!build.spec.revision.git.commit">
-                  <osc-git-link uri="build.spec.source.git.uri">{{build.spec.source.git.uri}}</osc-git-link>
-                </span>
-              </span>
-              <span ng-switch-default>
-                {{build.spec.source.type || 'Unknown'}}
-              </span>
-            </span>
-          </div>
-        </div>
-      </div>
+  <div class="pod-template-container">
+    <div class="pod-template-block" ng-repeat="container in initContainers | orderBy:'name'">
 
-      <div row ng-if="detailed && (container.command.length || container.args.length)" class="icon-row">
-        <div class="icon-wrap">
-          <span aria-hidden="true" class="fa fa-terminal"></span>
-        </div>
-        <div flex class="word-break">
-          <span class="pod-template-key">
-            Command:
-          </span>
-          <span>
-            <code class="command">
-              <truncate-long-text
-                  content="container | entrypoint : imagesByDockerReference[container.image]"
-                  limit="80"
-                  newline-limit="1"
-                  expandable="true"
-                  use-word-boundary="false">
-              </truncate-long-text>
-            </code>
-          </span>
-        </div>
-      </div>
+      <pod-template-container
+        pod-template-container="container"
+        images-by-docker-reference="imagesByDockerReference"
+        builds="builds"
+        detailed="detailed"
+        label-prefix="Init Container"></pod-template-container>
 
-      <div row ng-if="container.ports.length > 0" class="pod-template-ports icon-row">
-        <div class="icon-wrap">
-          <span data-icon="" aria-hidden="true" style="font-size:16px;line-height:normal"></span>
-        </div>
-        <div flex class="word-break">
-          <span class="pod-template-key">Ports:</span>
-          <span ng-repeat="port in container.ports | orderBy: 'containerPort' | limitToOrAll : detailed ? undefined : 1">
-            <span class="nowrap">{{port.containerPort}}/{{port.protocol}}</span><span ng-if="port.name"><span class="nowrap"> ({{port.name}})</span></span><span ng-if="port.hostPort"><span class="nowrap"><span class="port-icon"> &#8594;</span> {{port.hostPort}}</span></span><span ng-if="!$last">, </span>
-          </span>
-          <span ng-if="!detailed && container.ports.length >= 2">
-            and {{container.ports.length - 1}}
-            <span ng-if="container.ports.length > 2">others</span>
-            <span ng-if="container.ports.length === 2">other</span>
-          </span>
-        </div>
-      </div>
+      <div
+        extension-point
+        extension-name="container-links"
+        extension-types="link dom"
+        extension-args="[container, initContainers]"></div>
```

Comment:
Hm, can't decide if we want / need the extension point for init containers. There's no way for the extension to know that it's an init container, so I'm inclined to leave it out.

@jwforres